### PR TITLE
Implement Student Transfer Request

### PR DESCRIPTION
**Pull request by xFayre**
Implemented Logic to create a Transfer Request
Created Transfer Request Object

### CODE REVIEW

File: client/src/app/pages/transfer-class-group/transfer-class-group.page.ts [hunk 1]
```diff
@@ -40,6 +41,13 @@ export class TransferClassGroupPage implements OnInit {
     });
   }
 
+  requestClassGroupTransfer(transferRequest: TransferRequest) {
+    this.api.requestClassGroupTransfer(transferRequest).subscribe((data) => {
+      console.log("Transfer Request Successfull");
```
Comment (hvoliveira) [inline client/src/app/pages/transfer-class-group/transfer-class-group.page.ts]: Successful**

File: client/src/app/pages/transfer-class-group/transfer-class-group.page.ts [hunk 2]
```diff
@@ -57,10 +65,10 @@ export class TransferClassGroupPage implements OnInit {
           handler: () => {
             console.log('Confirm Okay');
             // this.cardClicked(classGroup);
```
Comment (hvoliveira) [inline client/src/app/pages/transfer-class-group/transfer-class-group.page.ts]: Why is this commented?

Comment (xFayre) [reply client/src/app/pages/transfer-class-group/transfer-class-group.page.ts]: Should have been deleted

File: client/src/app/pages/transfer-class-group/transfer-class-group.page.ts (hunk 1 above)
Comment (xFayre) [reply client/src/app/pages/transfer-class-group/transfer-class-group.page.ts]: Done

**Pull request merged**